auth.env.sample

```diff
@@ -0,0 +1,2 @@
+TEST_USER_UID=uid
+CLIENT_API_KEY=key
```

Comment:
This field is not used in production?

Reply:
No, CLIENT_API_KEY is only needed to retrieve an idToken for the test user. It's not optimal, but I guess it might make sense since the only reason idToken is needed is to verify authentication when communicating with a custom backend. But if anyone knows of a way to retrieve idTokens for tests without needing a client API key, that would be optimal.

Reply:
I have removed the part with authorization on firebase servers, and this variable is unused now. We can keep it for future integration tests